_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.info.yml

```diff
@@ -13,8 +13,6 @@ stylesheets-remove:
   - core/modules/system/css/system.module.css
   - core/modules/system/css/system.theme.css
   - core/modules/views/css/views.module.css
-ckeditor_stylesheets:
```

Comment:
Oh my gosh! Thank you for removing this in both themes. I kept meaning to create a JIRA issue for this exact task, but I forgot. Thanks :)